modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -0,0 +1,827 @@
+
+    # If your reads are unpaired you may want to run on slices to avoid
+    # making overlong rough models.  If you want to do this, specify a
+    # slice length here otherwise it will default to whole chromosomes.
+    slice_length => 10000000,
+
+    # Regular expression to allow FastQ files to be correctly paired,
+    # for example: file_1.fastq and file_2.fastq could be paired using
+    # the expression "\S+_(\d)\.\S+".  Need to identify the read number
+    # in brackets; the name the read number (1, 2) and the
+    # extension.
+    pairing_regex => '\S+_(\d)\.\S+',
+
+    # What Read group tag would you like to group your samples
+    # by? Default = ID
+    read_group_tag => 'SM',
+    read_id_tag    => 'ID',
+
+    use_threads => 3,
+
+    star_threads    => 12,
+    scallop_threads => 2,
+
+    # Please assign some or all columns from the summary file to the
+    # some or all of the following categories.  Multiple values can be
+    # separted with commas. ID, SM, DS, CN, is_paired, filename, read_length, is_13plus,
+    # is_mate_1 are required. If pairing_regex can work for you, set is_mate_1 to -1.
+    # You can use any other tag specified in the SAM specification:
+    # http://samtools.github.io/hts-specs/SAMv1.pdf
+
+    ####################################################################
+    # This is just an example based on the file snippet shown below.  It
+    # will vary depending on how your data looks.
+    ####################################################################
+    file_columns => [ 'SM', 'ID', 'is_paired', 'filename', 'is_mate_1', 'read_length', 'is_13plus', 'CN', 'PL', 'DS' ],
+
+    'filename_tag' => 'filename',    # For the analysis that creates star jobs, though I assume we should need to do it this way
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# No option below this mark should be modified
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+
+########################
+# db info
+########################
+    'reference_db' => {
+      -dbname => $self->o('reference_db_name'),
+      -host   => $self->o('reference_db_server'),
+      -port   => $self->o('reference_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'star_rnaseq_for_layer_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . 'star_rs_layer_' . $self->o('release_number'),
+      -host   => $self->o('star_rnaseq_for_layer_db_server'),
+      -port   => $self->o('star_rnaseq_for_layer_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'star_rnaseq_for_layer_nr_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_star_rs_layer_nr_' . $self->o('release_number'),
+      -host   => $self->o('star_rnaseq_for_layer_db_server'),
+      -port   => $self->o('star_rnaseq_for_layer_db_port'),
```

Comment:
Could you please use `star_rnaseq_for_layer_nr_db_*` instead of `star_rnaseq_for_layer_db_*`? This allows more flexibility in the servers. You will also need to add the values around lines 90 91

Reply:
Added.